src/wizards/clusters.ts

```diff
@@ -12,12 +12,18 @@ interface AddClusterState extends State {
     bootstrap: string;
     name: string;
     saslOption: SaslOption;
+    ssl: boolean;

```

Comment:
What about extending ConnectionOptions  ?

Reply:
it already extends State

Reply:
I mean:

```ts
interface AddClusterState extends State, ConnectionOptions {
    name: string;
}
```

Reply:
done